src/harness/testcases/testdata/fss_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "incumbent/ibfs/0",
+  "type": "FSS",
+  "deploymentParam": 
```

Comment:
I think Takashi's comment from SIQ.2 applies here as well: deploymentParam should be an array of DeploymentParam objects rather than a single object (same for the GWBL below). I'll try to make this clearer in the testing API doc -- I think it's my fault that it's not very clear right now.